scrubble-client/package-lock.json

```diff
@@ -1,8 +1,2814 @@
 {
   "name": "scrubble-client",
   "version": "1.0.0",
-  "lockfileVersion": 1,
```

Comment:
When this `package-lock` file is changing without making modifications to the `package.json` file it's probably an indication that we are using different versions of npm / node.

We should use something like [nvm](https://github.com/nvm-sh/nvm) to make sure we are using consistent tooling

Reply:
will attempt to set up nvm today